workflow/rules/bcftools_region_filter.smk

```diff
@@ -0,0 +1,77 @@
+# vim: syntax=python tabstop=4 expandtab
```

Comment:
I think we agreed on naming the `.smk`-files according to the tool so `bcftools.smk` 😜